runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -466,8 +474,34 @@ class CompilationInfo
          return std::get<0>(stream->read<bool>());
          }
 #endif /* defined(J9VM_OPT_JITSERVER) */
-      return (((uintptr_t)method->extra) & J9_STARTPC_NOT_TRANSLATED) == 0;
+
+      return (getPCIfCompiled(method) != NULL);
+      }
+
+   /**
+    * @brief Returns the PC of a method that is compiled
+    *
+    * @param method pointer to the J9Method
+    *
+    * @return The start PC if compiled, NULL otherwise
+    */
+   static void* getPCIfCompiled(J9Method *method)
+      {
+#if defined(J9VM_OPT_JITSERVER)
+      if (auto stream = getStream())
+         {
+         stream->write(JITServer::MessageType::CompInfo_isCompiled, method);
```

Comment:
shouldn't this be `MessageType::CompInfo_getPCIfCompiled` ?

Reply:
Damn, yeah you're right, thanks for catching it.